Gruntfile.js

```diff
@@ -32,7 +37,7 @@ module.exports = function(grunt) {
       build: {
         src: './build/ros3d.js',
         dest: './build/ros3d.min.js'
-      }
+      },
```

Comment:
Watch out for dangling comma... or maybe linter is fine with this

Reply:
Current jshint rules are unopinionated about dangling commas. After looking around I haven't seen dangling commas elsewhere so I'll stick to the defacto no-trailing-comma style for this codebase.